CMakeLists.txt

```diff
@@ -308,19 +308,19 @@ endif()
 # ==================================================================================================
 # Release compiler flags
 # ==================================================================================================
-if (NOT MSVC)
+if (NOT MSVC AND NOT IOS)
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fomit-frame-pointer")
```

Comment:
A comment here and for exceptions would explain why these flags are set up in this particular way (not explicitly about Crashlytics, but about stack unwinding in general).

Reply:
6bddd26c1efe66e41a9eca47feb622e54d4cfa9c